lago/virt.py

```diff
@@ -114,6 +115,51 @@ def _create_vm(self, vm_spec):
         vm_spec['vm-type'] = vm_type_name
         return vm_type(self, vm_spec)
 
+    def check_memory_requirements(self, conf):
```

Comment:
this is not so simple, now we have vm-providers, meaning that not all the vms might be on the local libvirt, so the required amount of memory is not the total sum of all of them (some might not even declare a memory entry, like the ones using the ssh provider)